src/go/coverage.txt

```diff
@@ -0,0 +1,122 @@
+mode: atomic
```

Comment:
extraneous file I think? (usually most folks make this coverage.out and I think it's ignored)